### PR TITLE
[VYMCA-33] [VYMCA-36] Latest videos/events if no featured results

### PR DESCRIPTION
**Pull request by hamrant**
**Steps for review:**

- [x] Login as admin
- [x] Create event series (check that you can see new required image field)
- [x] Save
- [x] Go to front page
- [x] Check that you can see 6 events
- [x] In teasers of events should be selected image
- [x] Check that you can 2 featured videos
- [x] Edit featured videos and disable featured checkbox
- [x] Return to front page
- [x] Check that you can see 6 latest videos (or less if there no 6 videos)
- [x] Edit several event instances
- [x] Set featured checkbox and update image (use different) on instance level
- [x] Return to front page - check that you can see only featured events
- [x] Go to events page (View all link)
- [x] Check that arrows in filter change content on the page and this content contains only selected date

### CODE REVIEW

File: modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+dependencies:
+  module:
+    - media
+    - recurring_events
+id: eventinstance.field_ls_image
+field_name: field_ls_image
+entity_type: eventinstance
+type: entity_reference
+settings:
+  target_type: media
```
Comment (AndreyMaximov) [inline modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml:12]: We need to limit it to Image bundle only

File: modules/openy_gc_storage/config/install/field.storage.eventseries.field_ls_image.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+dependencies:
+  module:
+    - media
+    - recurring_events
+id: eventseries.field_ls_image
+field_name: field_ls_image
+entity_type: eventseries
+type: entity_reference
+settings:
+  target_type: media
```
Comment (AndreyMaximov) [inline modules/openy_gc_storage/config/install/field.storage.eventseries.field_ls_image.yml:12]: We need to limit it to Image bundle only

File: modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml (hunk 1 above)
Comment (hamrant) [reply modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml:12]: Already limited here - https://github.com/fivejars/openy_gated_content/pull/30/files#diff-890ad3e5720a1a271a5ff4539678c6ddR28

File: modules/openy_gc_storage/config/install/field.storage.eventseries.field_ls_image.yml (hunk 2 above)
Comment (hamrant) [reply modules/openy_gc_storage/config/install/field.storage.eventseries.field_ls_image.yml:12]: Limited here - https://github.com/fivejars/openy_gated_content/pull/30/files#diff-a197f9ec7a697ac41c1c395bc70f6703R28

File: modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml (hunk 1 above)
Comment (hamrant) [reply modules/openy_gc_storage/config/install/field.storage.eventinstance.field_ls_image.yml:12]: Looks like there is a problem with Open Y entity browser views (it should be limited by image type)

**Pull request merged**